.github/workflows/ci-testing-deploy.yml

```diff
@@ -440,6 +440,56 @@ jobs:
           name: unit_postgresdb_coverage
           path: codeclimate.unit_postgresdb_coverage.json
 
+  unit-test-service-integration:
+    name: "[unit] service-integration"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python: [3.6]
```

Comment:
I know I've asked this again, but why are we adding a new service with version 3.6 when the we are trying to upgrade to 3.8?

Reply:
reminder: the main problem is that some of the libraries in the requirements.txt files cannot be installed in newer versions of python (e.g. backport libraries like ``dataclasses``) and pip-compile does not seem to respect constraints on python versions (BTW i should add a ticket on that).